compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/intermediate/PropertyLValue.kt

```diff
@@ -81,7 +81,8 @@ class FieldPropertyLValue(
     PropertyLValueBase(context, scope, startOffset, endOffset, origin, type, callReceiver, superQualifier) {
 
     override fun load(): IrExpression =
-        callReceiver.call { dispatchReceiverValue, extensionReceiverValue ->
+        // TODO: Use context receiver values
+        callReceiver.call { dispatchReceiverValue, extensionReceiverValue, _ ->
```

Comment:
`FieldPropertyLValue` is created for Java fields. Such properties should not have context receivers. Add an assertion here that `contextReceiverValues.isEmpty()`.